src/utils/getArticles.js

```diff
@@ -0,0 +1,41 @@
+const axios = require('axios');
+const sources = require('../data/sources.json');
+const sc = require('short-crypt');
+const getDataFromCheerio = require('./getDataFromCheerio');
+
+async function getArticles() {
+    const defaultImage =
+        'https://raw.githubusercontent.com/MizouziE/energy-prices-api/master/public/img/energy-prices-api-socials.png';
+    let articles = {};
+
+    sources.forEach(async (source) => {
+        try {
+            const response = await axios.get(source.site);
+            const returnedArticles = getDataFromCheerio(response.data);
+
+            returnedArticles.forEach(async (returned) => {
+                // now we have an array of multiple article's data to iterate through
+                articles[
+                    new sc(returned.url)
+                        .encryptToQRCodeAlphanumeric(returned.url)
+                        .slice(0, 10)
+                ] = {
+                    // use short-crypt to "hash" the url which works as unique ID
+                    title: returned.title,
+                    url: source.base + returned.url,
+                    source: source.name,
+                    image: /^((http|https):\/\/)/.test(returned.image)
+                        ? returned.image
+                        : defaultImage,
+                    region: source.region,
+                };
+            });
+        } catch (error) {
+            console.log({ error });
+            return;
+        }
+    });
```

Comment:
In the future we can add another console.log() here to show some feedback about the articles returned. Maybe just the object length and the date fetched.

Again, it's fine without, but something to consider down the line.